controllers/games.js

```diff
@@ -33,8 +33,20 @@ exports.getReviews = (req, res) => {
 };
 
 exports.getReviewById = (req, res, next) => {
-    selectReviewById(req.params.review_id).then((result) => {
-        res.status(200).send({review: result})
+    const reviewQuery = selectReviewById(req.params.review_id);
+    const commentQuery = selectCommentsByReviewId(req.params.review_id);
+    console.log('got to this point')
+
+    return Promise.all([reviewQuery, commentQuery])
+    .then((results) => {
+
+        const review = results[0];
+        const comments = results[1];
+
+        const comment_count = comments.length;
+        review['comment_count'] = comment_count;
+        
+        res.status(200).send({review: review})
     })
```

Comment:
We dont need to do 2 queries to get the comment count here, we can do something similar with the count function like when we get all the reviews

Reply:
just noticed, you are not using the count function when you get the comment count for the reviews, refactor both of these to a single query using the count function in SQL